### PR TITLE
Stripenew

### PR DESCRIPTION
**Pull request by alasalle**
# Description

Adds a stripe payment button. Will need to be refactored a bit to work with a seeded database and authenticated users. Right now, I'm creating a user and assigning a stripe payment subscription in the createSubcription resolver. Later, it will pull in existing user info.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change status

- [ ] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

I verified it worked with developers tools and checking the database. I will write tests for it.

# Checklist:

- [] My code follows the style guidelines of this project
- [] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

**Comment by alasalle** [comment]
[Implement Stripe](https://trello.com/c/AmjwicxE/45-implement-stripe)

File: client/src/components/stripe/stripe.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+const createSubscriptionMutation = gql`
+  mutation createSubscription($source: String!) {
+    createSubscription(source: $source) {
+      id
+      email
+    }
+  }
+`;
+
+const Stripe = ({ mutate }) => {
```
Comment (TheGameFreak720) [inline client/src/components/stripe/stripe.js]: This looks really clean. With one look I could figure out what it was about

File: client/src/index.js [hunk 2]
```diff
@@ -7,7 +7,8 @@ import { ApolloProvider } from 'react-apollo';
 import { BrowserRouter as Router } from 'react-router-dom';
 
 const client = new ApolloClient({
-	uri: 'https://labspt-diy-1bdc9280b0.herokuapp.com/labspt2-diy/dev',
+	// uri: 'https://labspt-diy-1bdc9280b0.herokuapp.com/labspt2-diy/dev',
+	uri: 'https://guarded-beach-26773.herokuapp.com/'
```
Comment (TheGameFreak720) [inline client/src/index.js]: Did you use the prisma one click deployment for this?

File: server/package.json [hunk 3]
```diff
@@ -3,15 +3,22 @@
...
   },
   "license": "MIT",
   "dependencies": {
+    "dotenv": "^6.2.0",
+    "global": "^4.3.2",
     "graphql-yoga": "^1.17.4",
     "node-dev": "^3.1.3",
-    "prisma-client-lib": "^1.27.4"
+    "prisma": "^1.28.3",
+    "prisma-binding": "^2.3.7",
```
Comment (TheGameFreak720) [inline server/package.json]: Did you actually get binding to work? Did that let you add custom mutation and Queries?

**Pull request merged**